x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_about_section.tsx

```diff
@@ -76,9 +76,15 @@ export const Author = ({ author }: AuthorProps) => (
   <BadgeList badges={author} data-test-subj="authorPropertyValue" />
 );
 
-export const BuildingBlock = () => (
+interface BuildingBlockProps {
+  isBuildingBlockEnabled: boolean;
```

Comment:
It could be changed to `type`. There is only one type available for now `default`.

```suggestion
  type: BuildingBlockType | undefined;
```